src/lib/SideBar.svelte

```diff
@@ -21,15 +20,31 @@
 		{ icon: 'fab fa-discord', info: 'Discord', activity: null },
 		{ icon: 'fab fa-github', info: 'GitHub', activity: null },
 	];
-
-	let activeInfo = null;
-
+	let activeInfo = null; // Tracks currently visible info
+	let lastHoveredInfo = null; // Tracks last hovered info
+	let isPanelHovered = false; // Tracks if info panel is hovered
+	let hideTimeout = null; // Timeout for hiding info panel
 	function showInfo(info) {
-		activeInfo = info;
+		clearTimeout(hideTimeout); // Cancel timeout
```

Comment:
We should reset hideTimeout to 0 to avoid clearing it multiple times. It might be also sensible to add an if(hideTimeout) before, unless we are sure the timeout is active